app/dashboard/templates/tests-board-job-kernel.html

```diff
@@ -0,0 +1,129 @@
+        display: block;
+        margin: 0 auto;
+    }
+</style>
+{%- endblock %}
+{%- block content %}
+<div class="row">
+    <div class="page-header">
+        <h3 id="details">{{ body_title|safe }}</h3>
+    </div>
+    <div class="col-xs-12 col-sm-12 col-md-7 col-lg-7">
+        <dl class="dl-horizontal">
+            <dt>Board</dt>
+            <dd class="loading-content" id="board">
+                <small>
+                    <i class="fa fa-circle-o-notch fa-spin fa-fw"></i>&nbsp;loading&hellip;
+                </small>
+            </dd>
+            <dt>Tree</dt>
+            <dd class="loading-content" id="tree">
+                <small>
+                    <i class="fa fa-circle-o-notch fa-spin fa-fw"></i>&nbsp;loading&hellip;
+                </small>
+            </dd>
+            <dt>Git branch</dt>
+            <dd class="loading-content" id="git-branch">
+                <small>
+                    <i class="fa fa-circle-o-notch fa-spin fa-fw"></i>&nbsp;loading&hellip;
+                </small>
+            </dd>
+            <dt>Git describe</dt>
+            <dd class="loading-content" id="git-describe">
+                <small>
+                    <i class="fa fa-circle-o-notch fa-spin fa-fw"></i>&nbsp;loading&hellip;
+                </small>
+            </dd>
+            <dt>Git URL</dt>
+            <dd class="loading-content" id="git-url">
+                <small>
+                    <i class="fa fa-circle-o-notch fa-spin fa-fw"></i>&nbsp;loading&hellip;
+                </small>
+            </dd>
+            <dt>Git commit</dt>
+            <dd class="loading-content" id="git-commit">
+                <small>
+                    <i class="fa fa-circle-o-notch fa-spin fa-fw"></i>&nbsp;loading&hellip;
+                </small>
+            </dd>
+            <dt>Date</dt>
+            <dd class="loading-content" id="job-date">
+                <small>
+                    <i class="fa fa-circle-o-notch fa-spin fa-fw"></i>&nbsp;loading&hellip;
+                </small>
+            </dd>
+        </dl>
+    </div>
+    <div class="col-xs-12 col-sm-12 col-md-5 col-lg-5">
+        <div id="pie-chart" class="chart-div pull-center">
+            <div id="pie-chart-heading">
+                <table id="pie-chart-legend" class="pie-chart">
+                    <tbody>
+                        <tr>
+                            <td id="success-cell" class="click-btn">0</td>
+                            <td>&nbsp;/&nbsp;</td>
+                            <td id="fail-cell" class="click-btn">0</td>
+                            <td>&nbsp;/&nbsp;</td>
+                            <td id="unknown-cell" class="click-btn">0</td>
+                        </tr>
+                    </tbody>
+                </table>
+            </div>
+            <div id="test-chart"></div>
+        </div>
+    </div>
+</div>
+{%- if lab_names|length > 0 %}
+<input type="hidden" id="lab-names" value="{{ lab_names|sort|join(',') }}">
```

Comment:
This is where the multiple `input`s should be done.
Inside the following for-loop, maybe added at the end of the for-loop.